src/codeu/chat/server/Controller.java

```diff
@@ -54,6 +56,10 @@ public ConversationHeader newConversation(String title, Uuid owner) {
     return newConversation(createId(), title, owner, Time.now());
   }
 
+  public UserInterest newUserInterest(String name) {
+    return newUserInterest(new HashSet<Uuid>(), new HashSet<Uuid>());
```

Comment:
Add parameter for user Uuid.
You shouldn't create a new UserInterest here, call `model.addUserInterest`.

The first `User` object you can get from `model.userById` and use the Uuid to get it. The second `User` model you can get from `model.userByText` and you can get using the name.